controller/instance_manager_controller.go

```diff
@@ -461,24 +462,41 @@ func (imc *InstanceManagerController) pollProcesses(im *longhorn.InstanceManager
 		if err != nil {
 			return err
 		}
-
-		updatedInstanceMap := make(map[string]types.InstanceProcessStatus)
-		for name, engine := range engines {
-			updatedInstanceMap[name] = engineToProcessStatus(engine)
+		// Will ignore the entries only in `engines`
```

Comment:
Confusing comment.

"Ignore the entries exists in `engines` but not in the `spec`."

Same below.